.github/workflows/build.yml

```diff
@@ -35,4 +35,4 @@ jobs:
           CONTRAST__API__API_KEY: ${{ secrets.CONTRAST__API__API_KEY }}
           CONTRAST__API__SERVICE_KEY: ${{ secrets.CONTRAST__API__SERVICE_KEY }}
           CONTRAST__API__ORGANIZATION_ID: ${{ secrets.CONTRAST__API__ORGANIZATION_ID }}
-        run: ./mvnw --batch-mode -Pend-to-end-test verify
```

Comment:
e2e testing was removed for this step, because the previous credentials used were out of date, causing this action to fail, and this repo is set to be archived anyway